keyboards/acheron/shark/rules.mk

```
@@ -0,0 +1,25 @@
+# MCU name
+MCU = STM32F303
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
```

No bootmagic at all?  Even bootmagic lite? 

```suggestion
BOOTMAGIC_ENABLE = yes     # Virtual DIP switch configuration
```

---

Updated